content/en/workflows/actions_catalog/_index.md

```diff
@@ -3,11 +3,16 @@ title: Actions Catalog
 kind: documentation
 type: actioncatalog
 disable_toc: true
-further_reading:
-- link: "logs/processing/pipelines"
-  tag: "Documentation"
-  text: "Log processing pipelines"
 cascade:
   disable_sidebar: true
   type: actioncatalog
 ---
+
+<div class="alert alert-warning">Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>
```

Comment:
Try using the {{ beta-callout }} shortcode.

Reply:
Are you sure? I had copied this from your doc to be consistent:
https://github.com/DataDog/documentation/blob/baf2c3e5072d9034d19678e8548fdd6f80554fb9/content/en/workflows/setup.md?plain=1#L11

I tried using the shortcode in a local build, and it looked pretty strange. Screenshot:
https://a.cl.ly/bLu5ZrDe

Removing all of the optional parameters listed in [docs4docs](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Beta-callouts) and adding `btn_hidden="true"` didn’t make any change to the appearance. This is the beta callout I ended up with, after initially starting with all of the parameters, and then removing them to try to alter the appearance:
```
{{< beta-callout url="#" btn_hidden="true" >}}
  Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
{{< /beta-callout >}}
```
Maybe I’m using it incorrectly — if so, let me know if you can spot my mistake! Otherwise, I’m guessing it may be helpful to follow up on #websites for this?